compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -1907,6 +1913,31 @@ private void visitActionInvocationEndpoint(BLangInvocation iExpr) {
         iExpr.expr = getClientExpr;
     }
 
+    /**
+     * Visit builtin function invocations.
+     * @param iExpr invocation expression
+     */
+    private void visitBuiltinFunctionInvocation(BLangInvocation iExpr) {
+        // Check if the builtin operation is 'length'
+        if (Constants.BUILTIN_LENGTH_OPERATION.equals(iExpr.name.value)) {
+            DiagnosticPos pos = iExpr.pos;
+            final BLangUnaryExpr lengthExpr = (BLangUnaryExpr) TreeBuilder.createUnaryExpressionNode();
+            BOperatorSymbol bOperatorSymbol = new BOperatorSymbol(names.fromString(OperatorKind.LENGTHOF.value()),
+                                                                  symTable.rootPkgSymbol.pkgID,
```

Comment:
I think this will look better if we assign each of these parameters to new variables above and then pass the variables when we create a new `BOperatorSymbol` instance.

Reply:
Fixed with 96af83f